packages/eslint-config-react/rules/react.js

```diff
@@ -17,5 +17,9 @@ module.exports = {
         ignoreFunctionalComponents: true,
       },
     ],
+    // Next.JS does not require JSX to be in scope
+    'react/react-in-jsx-scope': 'off',
+    // We're okay with this but use it sparingly and only when this is intentional
+    'react/jsx-props-no-spreading': 'off',
```

Comment:
I did notice the `explicitSpread` option that is set by default: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md#explicitspread. However, you probably just want to allow spreading in general. If so, then this is good.

Reply:
It is intentional. If we're going to wrap Material-UI components, the "dump of props" to the wrapped component is usually a good way to keep your wrapper usable.

Reply:
It's also nice to be able to do something like:

```
entities.map(entity => <EntityItem {...entity} />)
```

Reply:
Not opposed, just making you aware of that other option before we push the button :).